app/assets/javascripts/modules/tabs.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+moj.Modules.tabs = {
+  listSelector: 'ul.tabs',
+  panelSelector: 'section.tab-panel',
+  activeClass: 'active',
+
+  init: function() {
+    var self = this;
+
+    self.$list = $(self.listSelector);
+
+    if(self.$list.length) {
+      self.$tabs = self.$list.find('li');
+      self.$panels = $(self.panelSelector);
+
+      self.storeSlugs();
+      self.showInitialTab();
+      self.bindEvents();
+    }
+  },
+
+  bindEvents: function() {
+    var self = this;
+
+    self.$list.on('click', function(e) {
+      self.selectTab($(e.target));
+    });
+  },
+
+  showInitialTab: function() {
```

Comment:
I get what this is doing and it seems to mostly work as well, but when I refresh a page it doesn't go to the right tab as expected (despite the URL having the hash component)

Reply:
Hmm, odd. Are you using some crazy minority browser like Safari?

Reply:
Nope, ignore me, that was me playing with your code and breaking it, sorry Clive 😭

Reply:
👍